src/bitcoin-cli.cpp

```diff
@@ -37,6 +37,7 @@ std::string HelpMessageCli()
     strUsage += HelpMessageOpt("-?", _("This help message"));
     strUsage += HelpMessageOpt("-conf=<file>", strprintf(_("Specify configuration file (default: %s)"), BITCOIN_CONF_FILENAME));
     strUsage += HelpMessageOpt("-datadir=<dir>", _("Specify data directory"));
+    strUsage += HelpMessageOpt("-getinfo", _("Get general information from the remote server"));
```

Comment:
Are you going to update the help text to note something about balance/height atomicity? The old getinfo takes cs_main and cs_wallet and returns atomic information, this no longer does. getwalletinfo will likely eventually return something so that you can report if a result was atomic here (probably a follow-up to #10286).

Reply:
What do you suggest that the text for such a note be?

Reply:
as getinfo is misleading here too, see https://github.com/bitcoin/bitcoin/pull/10871#issuecomment-324699582, I still don't see this as a strong point, but as it seems to bother @TheBlueMatt very much let's just add it...

Reply:
@laanwj I do not believe that to be the case in any release nor on master. It will be the case after #10286, however there is no getinfo on master either....